terraform/knimer/slack/variables.tf

```diff
@@ -3,8 +3,13 @@ variable "name_prefix" {
   type        = string
 }
 
+variable "slack_webhook_url_secret_name" {
+  description = "Name of the parameter containing the slack webhook url"
+  type        = string
+}
+
 variable "slack_webhook_url_secret_arn" {
```

Comment:
does the `lambda` module need `slack_webhook_url_secret_arn`?

Reply:
it's still needed for the `lambda-secret-access-policy`